packages/golden-layout/.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules
```

Comment:
Doing an `npm install` or `npm run clean` causes a bunch of tracked changes:

```
Changes not staged for commit:
  (use "git add/rm <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        deleted:    packages/golden-layout/dist/goldenlayout.js
        deleted:    packages/golden-layout/dist/goldenlayout.min.js
        modified:   packages/golden-layout/package-lock.json
        deleted:    packages/golden-layout/src/css/goldenlayout-base.css
        deleted:    packages/golden-layout/src/css/goldenlayout-base.css.map
        deleted:    packages/golden-layout/src/css/goldenlayout-dark-theme.css
        deleted:    packages/golden-layout/src/css/goldenlayout-dark-theme.css.map
        deleted:    packages/golden-layout/src/css/goldenlayout-light-theme.css
        deleted:    packages/golden-layout/src/css/goldenlayout-light-theme.css.map
        deleted:    packages/golden-layout/src/css/goldenlayout-soda-theme.css
        deleted:    packages/golden-layout/src/css/goldenlayout-soda-theme.css.map
        deleted:    packages/golden-layout/src/css/goldenlayout-translucent-theme.css
        deleted:    packages/golden-layout/src/css/goldenlayout-translucent-theme.css.map
```

Reply:
Test this again. I think it should be fixed